bombastic/api/src/server.rs

```diff
@@ -73,15 +74,18 @@ enum Error {
     InvalidContentEncoding,
 }
 
+#[allow(clippy::unwrap_used)]
```

Comment:
I don't love this

Reply:
That's an example for when you need to opt-out. Which is ok sometimes I guess.